examples/profile-client/src/main/java/com/palantir/example/profile/schema/generated/UserPhotosStreamStore.java

```diff
@@ -219,7 +219,7 @@ protected void loadSingleBlockToOutputStream(Transaction t, Long streamId, long
                 Long streamId = v.getColumnName().getStreamId();
                 Sha256Hash hash = r.getHash();
                 if (hashForStreams.containsKey(streamId)) {
-                    AssertUtils.assertAndLog(log, hashForStreams.get(streamId).equals(hash), "(BUG) Stream ID has 2 different hashes: " + streamId);
+                    AssertUtils.assertAndLog(hashForStreams.get(streamId).equals(hash), "(BUG) Stream ID has 2 different hashes: " + streamId);
```

Comment:
I guess we added this logger recently, we might have added it to the gen code but not to schema renderer, can you add it to the schema renderer?